index.js

```diff
@@ -27,6 +27,8 @@ async function run() {
     const taskDefinitionFamily = core.getInput('task-definition-family', { required: false }) || undefined;
     const taskDefinitionRevision = Number(core.getInput('task-definition-revision', { required: false })) || null;
 
```

Comment:
nit: extra line

Reply:
Fixed the extra line. Resolving the comment.